db/people.yml

```diff
@@ -1492,10 +1492,27 @@ gpaglia:
   position_full: Senior Project Manager - Analytics
 hcchiu:
   active: true
-  aka: []
-  avatar: https://github.com/jasonchiu1997.png
-  bio: bio
-  education: []
+  aka:
+    - Jason Chiu
+    - Jason
+    - Hoi Chun Chiu
+    - Hoi Chun
+  avatar: https://github.com/jasonhcc.png
+  bio: Hoi Chun is an Applied Physics PhD student from Hong Kong. Prior to joining Columbia, he obtained a BSc
+    in Physics from HKUST. He joined the Billinge group in summer 2021. He loves listening to Cantonese music
+    and watching movies during his free time.
+  education:
+    - begin_year: 2016
+      begin_month: 9
+      degree: BSc in Physics
+      institution: Hong Kong University of Science & Technology
+      end_year: 2020
+      end_month: 7
+    - begin_year: 2020
```

Comment:
failing tests because there is a problem with indentation here

Reply:
I saw the problem here. I will push an updated version in a few minutes.

Reply:
@sbillinge I have deleted the indentation.